### DISCUSSION TITLE
custom command documentation

### DISCUSSION THREAD

**Discussion by mvolkmann** [nushell/nushell#3181] [Q&A]
Here's a simple example of a custom command with documentation:

```nu
# Computes the sum of two numbers.
def sum [
  n1: number, # first number
  n2: number # second number
] {
  = $n1 + $n2
}
```

When I enter `help sum` it outputs the following:

```text
Usage:
  > sum <n1> <n2> {flags}

Parameters:
  <n1> first number
  <n2> second number

Flags:
  -h, --help: Display this help message
```

Why do I not see the documentation from the comment above the `def` line?

**Comment by fdncred** [comment]
It seems to work on Windows. Are you typing this command into the prompt or are you sourcing a script?

![image](https://user-images.githubusercontent.com/343840/111311720-8d2aaf80-862c-11eb-8435-ff30357966dd.png)

**Reply by ghost** [reply]
This works for me as well on macOS nu version 0.30.
<img width="365" alt="Screen Shot 2021-05-07 at 11 53 06 AM" src="https://user-images.githubusercontent.com/472721/117476123-d4496880-af2a-11eb-9117-abbc24ead2dd.png">

**Reply by ghost** [reply]
`help sum` is equivalent.
